backend/invoke_ai_web_server.py

```diff
@@ -1564,6 +1571,19 @@ def dataURL_to_image(dataURL: str) -> ImageType:
     )
     return image
 
+"""
+Converts an image into a base64 image dataURL.
+"""
+
+def image_to_dataURL(image):
+    buffered = io.BytesIO()
+    image.save(buffered, format="PNG")
+    image_base64 = "data:image/png;base64," + base64.b64encode(
+        buffered.getvalue()
+    ).decode("UTF-8")
+    return image_base64
+
+
```

Comment:
```suggestion
"""
Converts an image into a base64 image dataURL.
"""

def image_to_dataURL(image: ImageType) -> str:
    buffered = io.BytesIO()
    image.save(buffered, format="PNG")
    image_base64 = "data:image/png;base64," + base64.b64encode(
        buffered.getvalue()
    ).decode("UTF-8")
    return image_base64


```

Just adding types here for clarity (I have imported the PIL Image type as ImageType).